### DISCUSSION TITLE
How to set env to UAT when use HTTP API Client?

### DISCUSSION THREAD

**Discussion by Lingzhou314** [apolloconfig/apollo#5235] [Q&A] [answered]
URL: {config_server_url}/configs/{appId}/{clusterName}/{namespaceName}?releaseKey={releaseKey}&messages={messages}&label={label}&ip={clientIp}

the url has no "env" variable

i added query param "&env=UAT" , but no effect, return DEV config again.

**Comment by nobodyiam** [comment, marked as answer]
different env has different {config_server_url}
